explorer_relevance_map:
  relevance_map:
    ros__parameters:
      base_increment_rate: 0.03
      camera_info_topic: /camera_info
      debug_mode: true
      decay_factor: 0.999
      fov_deg: 70.0
      frame_map: map
      frame_robot: base_link
      input_graph_topic: /exploration_graph_nodes/graph_nodes
      input_map_topic: /map
      max_range: 5.0
      output_graph_topic: /filtered/exploration_graph_nodes/graph_nodes
      qos_overrides./clock.subscription.depth: 1
      qos_overrides./clock.subscription.durability: volatile
      qos_overrides./clock.subscription.history: keep_last
      qos_overrides./clock.subscription.reliability: best_effort
      qos_overrides./parameter_events.publisher.depth: 1000
      qos_overrides./parameter_events.publisher.durability: volatile
      qos_overrides./parameter_events.publisher.history: keep_last
      qos_overrides./parameter_events.publisher.reliability: reliable
      raytracing_enabled: true
      relevance_threshold: 0.8
      robot.frame_map: map
      robot.frame_robot: base_link
      angular_confidence_sharpness: 0.35
      radial_confidence_sharpness: 0.7
      ignore_high_score_threshold: 1.0

exploitation_relevance_map:
  relevance_map:
    ros__parameters:
      base_increment_rate: 0.03
      camera_info_topic: /camera_info
      debug_mode: true
      decay_factor: 0.99
      fov_deg: 70.0
      frame_map: map
      frame_robot: base_link
      input_graph_topic: /exploitation_graph_nodes/graph_nodes
      input_map_topic: /map
      max_range: 5.0
      output_graph_topic: /filtered/exploitation_graph_nodes/graph_nodes
      qos_overrides./clock.subscription.depth: 1
      qos_overrides./clock.subscription.durability: volatile
      qos_overrides./clock.subscription.history: keep_last
      qos_overrides./clock.subscription.reliability: best_effort
      qos_overrides./parameter_events.publisher.depth: 1000
      qos_overrides./parameter_events.publisher.durability: volatile
      qos_overrides./parameter_events.publisher.history: keep_last
      qos_overrides./parameter_events.publisher.reliability: reliable
      raytracing_enabled: false
      relevance_threshold: 0.8
      robot.frame_map: map
      robot.frame_robot: base_link
      angular_confidence_sharpness: 0.2
      radial_confidence_sharpness: 0.5
      ignore_high_score_threshold: 1.0

detection_relevance_map:
  relevance_map:
    ros__parameters:
      angular_confidence_sharpness: 0.1
      base_increment_rate: 0.015
      camera_info_topic: /camera_info
      debug_mode: true
      decay_factor: 0.999
      fov_deg: 70.0
      frame_map: map
      frame_robot: base_link
      ignore_high_score_threshold: 0.8
      input_graph_topic: /detection_graph_nodes/graph_nodes
      input_map_topic: /map
      max_range: 3.5
      output_graph_topic: /filtered/detection_graph_nodes/graph_nodes
      prompt_topic: /user_prompt
      qos_overrides./clock.subscription.depth: 1
      qos_overrides./clock.subscription.durability: volatile
      qos_overrides./clock.subscription.history: keep_last
      qos_overrides./clock.subscription.reliability: best_effort
      qos_overrides./parameter_events.publisher.depth: 1000
      qos_overrides./parameter_events.publisher.durability: volatile
      qos_overrides./parameter_events.publisher.history: keep_last
      qos_overrides./parameter_events.publisher.reliability: reliable
      radial_confidence_sharpness: 0.5
      raytracing_enabled: false
      relevance_threshold: 1.0
      robot.frame_map: map
      robot.frame_robot: base_link